src/wait-for-dom-change.js

```diff
@@ -0,0 +1,37 @@
+import 'mutationobserver-shim'
+
+function waitForDomChange({
+  container = document,
+  timeout = 4500,
+  mutationObserverOptions = {
+    subtree: true,
+    childList: true,
+    attributes: true,
+    characterData: true,
+  },
+} = {}) {
+  return new Promise((resolve, reject) => {
+    // Disabling eslint prefer-const below: either prefer-const or no-use-before-define triggers.
+    let lastError, observer, timer // eslint-disable-line prefer-const
```

Comment:
I'm pretty sure that if you remove `observer` and `timer` from here and use a `const` declaration below you should be fine :)